pkgs/by-name/en/ente-auth/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}:
+let
+  pname = "ente-auth";
+  version = "3.0.18-beta.2";
+
+  src = fetchurl {
+    url = "https://github.com/ente-io/ente/releases/download/auth-v${version}/ente-auth-v${version}-x86_64.AppImage";
+    hash = "sha256-WtFoqW++U+BoDUW1lc6xMJDCC1r2+O/fci492P2iVDg=";
+  };
+in
+
+appimageTools.wrapType2 {
+  inherit pname version src;
+  extraPkgs = pkgs: [
+    pkgs.libepoxy
+    pkgs.brotli
+  ];
+
+  meta = {
```

Comment:
Missing `platforms = [ "x86_64-linux" ];` (as the provided appimage only supports x86_64-linux)